### PR TITLE
typo fixes

### PR DESCRIPTION
**Pull request by DJSethDuncan**
## Summary

There were a handful of typos in various pages. This PR is solely to fix copy.

## Related issue(s)

Issue #25324 

## Testing done

The old code showed incorrect text in the ui, this fixes that text.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

I can't get the UI to spin up in codespaces properly but I suspect this is a low-risk change that doesn't _require_ screenshots. I understand if the reviewer disagrees and is unable to approve this PR, though.

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

This affects the text on 6 different confirmation pages.

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

I'm making an effort to contribute to various open source projects and the VA project seems cool. I'm hoping this PR helps but codespaces wasn't playing ball. Hoping this low-risk change is acceptable.

### CODE REVIEW

File: script/github-actions/daily-product-scan/tests/mocks/applications/app-1/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -102,8 +102,8 @@ export class ConfirmationPage extends React.Component {
           </p>
           <p>
             Please don’t apply again. Instead, please call our toll-free hotline
-            at <va-telephone contact="877-222-8387" />. We’re here Monday
```
Comment (adamwhitlock1) [inline script/github-actions/daily-product-scan/tests/mocks/applications/app-1/containers/ConfirmationPage.jsx:105]: you can actually import this number from a constants file that is maintained in the component-library, which I think is the 'official' way

``` js
import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';

// usage
<va-telephone contact={CONTACTS['222_VETS']} />
```

**Review by adamwhitlock1 — APPROVED** [review]
one minor thing, but non-blocking

File: script/github-actions/daily-product-scan/tests/mocks/applications/app-1/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (DJSethDuncan) [reply script/github-actions/daily-product-scan/tests/mocks/applications/app-1/containers/ConfirmationPage.jsx:105]: Oh weird, I didn't intend to change those and didn't catch it in the diff. I wonder if my IDE did something funky with formatting. I'll fix it in a commit shortly. Thanks!

**Review by rsmithadhoc — APPROVED** [review]

**Review by christinec-fftc — APPROVED** [review]

**Comment by DJSethDuncan** [comment]
Closing due to lack of required-reviewers giving a review after a long wait period. Have a great day!

**Pull request closed without merging**